### DISCUSSION TITLE
Chipolo android find my device trackers - any experience?

### DISCUSSION THREAD

**Discussion by cromelex** [ESPresense/ESPresense#1313] [General]
Hi all,

Just wondering if anyone has tried one of the new Chipolo (or similar) android Find my Device tracker successfully with ESPRESENSE?

I've been using Tile Stickers for the past 2 years on my cats, with very good results with ESPRESENSE. 
However, their built in battery is about to die, and when the cats escape out of the house, the Tile network isn't the best. 

Just wondering if anyone has found any of these newer tags to work with ESPRESENSE - I'd love to keep the same great espresense functionality and to potentially avail of the better network effect. 

If there's not feedback I might have just to chance it and try them myself

**Comment by cromelex** [comment]
So yeah that doesn't work. The Chipolo One Point only seems to have a random beacon that changes every 15 minutes and doesn't emit regularly enough to be even useful in any way. 

Info for anyone that looks at this in the future.

**Reply by BoldNZ** [reply]
Thanks for the update, I've been looking at some trackers for my dogs and was wondering if the Chipolo trackers would work.  I don't really want to get iBeacons..

**Reply by cromelex** [reply]
> Thanks for the update, I've been looking at some trackers for my dogs and was wondering if the Chipolo trackers would work. I don't really want to get iBeacons..

I bought a MotoTag this week as well, give me a week and I will be able to check if it is usable for Espresense.

**Comment by cromelex** [comment]
Ok so I initiated my moto tag today, and with a BLE Scanner on my phone I can see it identified as a "moto finder". The MAC seems to have not changed over the last  100 minutes. If this is actually static then the moto tag trackers might actually be usable with ESPresense, which would be great.

**Comment by cromelex** [comment]
Good news, my moto tag bluetooth mac hasn't changed and it's been over 24h already.
Will set it up with espresense and test further!

**Comment by cromelex** [comment]
Might need to open a separate discussion for the Moto Tag as this isn't straightforward at all. 

My pixel 7 shows a mac address under the Bluetooth settings, a different one with a BLE Scanner app (which also appears to be static), but neither of those seems to be visible on my espresense sensors. I have no clue how to work out the fingerprinting for this.
